src/components/GameDetail/GameProgress.vue

```diff
@@ -102,3 +103,116 @@ export default {
   },
 };
 </script>
+<style lang="scss" rel="stylesheet/scss" scoped>
+  @import "~styles/styles";
+
+  .progress-range {
```

Comment:
can you put these styles in `src/styles/_inputs.scss`? also, you can target all range inputs directly without a class if you want to:

```suggestion
  input[type="range"]
```